rule/struct_tag.go

```diff
@@ -102,6 +102,7 @@ const (
 	keyJSON     = "json"
 	keyProtobuf = "protobuf"
 	keyRequired = "required"
+	keyURL      = "url"
```

Comment:
I'd maybe update the docs to include all supported struct tags explicitly.

Reply:
Yes, this PR is one in a series of PR related to `struct-tag`, I'll will update the doc at the end.